pom.xml

```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.swisspush</groupId>
     <artifactId>rest-storage</artifactId>
-    <version>2.4.17-SNAPSHOT</version>
+    <version>3.5.0-SNAPSHOT</version>
```

Comment:
According to semver, we should jump to 2.5.0 as there is no change in our API.

Reply:
The idea was to express compatibility with Vert.x 3.5.0
(Note that I changed version for RedisQues accordingly: https://github.com/swisspush/vertx-redisques/blob/develop/pom.xml#L5 - probably you didn't see it)

Reply:
Unfortunately, this is not sustainable, because if we keep being compatible with vertx 3.6 or even 4.1, we don't convey something meanfingful anymore. Additionally, if we refactor the API in an incompatible way, it should semverly bump to 4.0, with compatibility with vertx still being 3.5. Also, this does not scale if we have more than one dependency (vertx and another one), which is the one that wins? In short, library versioning rarely depend on the dependencies.

Reply:
ok. I change to 2.5.0 and I'll do this also for RedisQues. PRs are coming